ipad.xcodeproj/project.pbxproj

```diff
@@ -2011,10 +1990,10 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				MARKETING_VERSION = 1.7;
+				MARKETING_VERSION = 2.1;
 				PRODUCT_BUNDLE_IDENTIFIER = ca.bc.gov.InvasivesBC;
 				PRODUCT_NAME = Inspect;
-				PROVISIONING_PROFILE_SPECIFIER = "";
+				PROVISIONING_PROFILE_SPECIFIER = "InvasivesBC Muscles";
```

Comment:
Just confirming that this is not a typo on our part - doesn't really matter if that's how it's spelled as long as it's working.

Reply:
ya. unfortunately, i think thats how the app was listed, so the profile that was issued to us was the same.